client/src/components/App.js

```diff
@@ -8,48 +8,54 @@ import Auth from './auth/Auth';
 import Home from './home/Home';
 import Dashboard from './dashboard/Dashboard';
 import ProfileInfoEdit from './profileInfoEdit/ProfileInfoEdit';
+import Editor from './editor/Editor'
+import Sidebar from './layout/Sidebar';
 
 import Register from './auth/Register';
 import Login from './auth/Login';
+import Layout from './Layout';
+import SidebarData from './layout/SidebarData'
 import Header from './layout/Header';
-import Sidebar from './layout/Sidebar';
 import { Container, Col, Row } from 'react-bootstrap';
 
 import './layout/Layout.css'
-
 import PublicationPage from './publications/PublicationPage';
 import { Fragment } from 'react';
```

Comment:
You might need to remove the unused import so that the Static Analyzer wont show error messages.

Reply:
Agree with message above, please pull the latest Logiciel branch to fix the conflicts.

Reply:
Yep I have deleted them, sorry my bad XP